backend/open_api_spec.yml

```diff
@@ -0,0 +1,830 @@
+                    enum: [ READ, WRITE ]
+                  published_at:
+                    type: number
+                  description:
+                    type: string
+                  publisher_metadata:
+                    $ref: "#/components/schemas/publisher_metadata"
+                  tombstone:
+                    type: boolean
+        "401":
+          $ref: "#/components/responses/401"
+        "404":
+          $ref: "#/components/responses/404"
+    put:
+      summary: Update a Collection's metadata.
+      description: >-
+        Start a revision of a published Collection or update its existing revision. Update a private Collection
+        in-place. If any Collection metadata fields are included in the body of the request, these fields SHALL be
+        updated for the specified Collection.
+      security:
+        - cxguserCookie: []
+        - curatorAccess: []
+      parameters:
+        - $ref: "#/components/parameters/path_collection_uuid"
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/collection_form_metadata"
+            examples:
+              update_name:
+                summary: Update the name
+                value:
+                  name: "New Collection name"
+              update_description:
+                summary: Update the description
+                value:
+                  description: "New Collection description"
+              start_new_revision:
+                summary: Start revision without making updates
+                value: {}
+
+      responses:
+        "200":
+          $ref: "#/components/responses/200"
+        "201":
+          $ref: "#/components/responses/201"
+        "400":
+          $ref: "#/components/responses/400_multiple"
+        "401":
+          $ref: "#/components/responses/401"
+        "404":
+          $ref: "#/components/responses/404"
+
+  /curation/collections/{collection_uuid}/datasets:
+    delete:
+      summary: Delete a private Dataset
+      description: >-
+        Delete a Dataset. The Dataset MUST NOT belong to a published Collection. MUST indicate the Dataset by providing
+        *one* of either the **curator_tag** (non-null) OR the **dataset_uuid** parameters.
+      security:
+        - cxguserCookie: []
+        - curatorAccess: []
+      parameters:
+        - $ref: "#/components/parameters/path_collection_uuid"
+        - $ref: "#/components/parameters/query_curator_tag"
+        - $ref: "#/components/parameters/query_dataset_uuid"
+      responses:
+        "202":
+          description: Accepted
+          content:
+            application/json:
+              schema:
+                type: string
+        "400":
+          $ref: "#/components/responses/400"
+        "401":
+          $ref: "#/components/responses/401"
+        "403":
+          $ref: "#/components/responses/403"
+        "404":
+          $ref: "#/components/responses/404"
+    patch:
```

Comment:
Should updating the curator_tag require that the Collection not be public?

When just considering the data involved and whether it is "okay" to update tags, which are not surfaced to other users, it seems like it would be fine to all the tags for a public Collection to be modified. However, I'm not sure if allowing the `dataset.updated_at` to be greater/after the `project.revised_at` date for its Collection might cause logical errors elsewhere in the pipeline.

Reply:
Per the table in [The data model must be updated to support filtering by date ranges](https://github.com/chanzuckerberg/single-cell-data-portal/issues/1339), updating collection *information* does not update the timestamps in question:

| Operation | collection published_at | collection revised_at | dataset published_at | dataset revised_at |
|:---:|:---:|:---:|:---:|:---:|
| Publish private collection | X | - | X | - |
| Update a dataset in revision | -  | X | -  | X |
| Add a dataset in revision | - | X | X | - |
| Delete a dataset in revision | - | X | - | - |
| Update collection information in a revision | - | -  | - | - |

Tags would be a similar case.